code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -55,6 +55,7 @@
 
 	//Hot simple_animal baby making vars
 	var/childtype = null
+	var/rarechildtype = null	//Special children
```

Comment:
Weighted list 1: Pure Byond:
```var/list/L = list(prob(prob_1); element_1, prob(prob_2); element_2)```
this uses regular ```pick()```

Weighted list 2: tgcode:
```var/list/L = list(element_1 = prob_1, element_2 = prob_2)```
this uses ```pickweight()```